src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+function convertToNumber(salaryString) {
+  return salaryString.slice(1).split(',').join('');
+}
+
+function sortList(list) {
+  const ul = document.querySelector('ul');
+  const employeesData = [...list]
+    .sort((person1, person2) =>
+      convertToNumber(person2.getAttribute('data-salary'))
+      - convertToNumber(person1.getAttribute('data-salary'))
+    );
+
+  return ul.append(...employeesData);
+}
+
+function getEmployees(list) {
+  const employees = [...list]
+    .map(person => {
+      const { position, salary, age } = person.dataset;
+      const employee = {};
+
+      employee.name = person.innerText;
+      employee.position = position;
+      employee.salary = salary;
+      employee.age = age;
```

Comment:
Why do you need an empty object here?

```suggestion
      const employee = {
        name: person.innerText,
        position,
        salary,
        age
      };
```